client/db/benches/state_access.rs

```diff
@@ -25,10 +25,39 @@ use sp_runtime::{
 	testing::{Block as RawBlock, ExtrinsicWrapper, Header},
 	StateVersion, Storage,
 };
+use std::time::Duration;
 use tempfile::TempDir;
 
 pub(crate) type Block = RawBlock<ExtrinsicWrapper<u64>>;
 
+fn get_polkadot_storage_keys() -> Vec<Vec<u8>> {
+	// This is a dump of all of the keys from Polkadot block
+	// `ad2fedfc2e6da47289b41cc24617915d4bce0638ca41bd0c4e4864ebb869f2dd`.
+	let path: std::path::PathBuf = env!("CARGO_MANIFEST_DIR").into();
+	let compressed_blob = std::fs::read(path.join("benches").join("keys.bin.zst")).unwrap();
+	let blob = zstd::bulk::decompress(&compressed_blob, 153917787).unwrap();
+	let keys: Vec<Vec<u8>> = codec::Decode::decode(&mut &blob[..]).unwrap();
+	assert_eq!(keys.len(), 1872246);
+	keys
+}
+
+fn get_account_keys(all_keys: &[Vec<u8>]) -> Vec<Vec<u8>> {
+	let prefix = [
+		0x26, 0xaa, 0x39, 0x4e, 0xea, 0x56, 0x30, 0xe0, 0x7c, 0x48, 0xae, 0x0c, 0x95, 0x58, 0xce,
+		0xf7, 0xb9, 0x9d, 0x88, 0x0e, 0xc6, 0x81, 0x79, 0x9c, 0x0c, 0xf3, 0x0e, 0x88, 0x86, 0x37,
+		0x1d, 0xa9,
+	];
```

Comment:
Adding some assert that there are actually keys with this prefix would be nice and a comment saying what the current prefix is.